silkworm/silkrpc/core/block_reader.cpp

```diff
@@ -73,7 +73,7 @@ Task<void> BlockReader::read_balance_changes(BlockCache& cache, const BlockNumbe
     co_return;
 }
 
-Task<void> BlockReader::load_addresses(uint64_t block_number, BalanceChanges& balance_changes) const {
+Task<void> BlockReader::load_addresses(BlockNum block_number, BalanceChanges& balance_changes) const {
```

Comment:
The same change should be applied also in `block_reader.hpp`

Reply:
ok